### PR TITLE
Adds 'stolen tech' category to nuke op uplink and some new items.

### PR DESCRIPTION
**Pull request by ghost**
What the fuck does the syndicate do with all that stolen tech? Use it to blow up more NT stations, duh!

This is finished and tested.

### CODE REVIEW

**Comment by ghost** [comment]
Also someone sprite up a cool syndicate flamethrower. In order for it to work correctly, the nozzle and tank attachment locations have to be in the same place on the sprite, so I suggest you 'build' around the current flamethrower.

**Comment by Neerti** [comment]
> [WIP] Adds 'stolen tech' category to nuke op uplink and some new items.
> Adds 'stolen tech' category to nuke op uplink
> nuke op uplink
> nuke op

10/10 danno.

**Comment by ghost** [comment]
I almost want to make the syndie flamethrower use less plasma somehow and add it to traitor uplink but then again I'm not sure how that code works

File: code/game/objects/items/weapons/RCD.dm [hunk 1]
```diff
@@ -261,4 +261,15 @@ RCD
 	anchored = 0.0
 	origin_tech = "materials=2"
 	m_amt = 16000
-	g_amt = 8000
+	g_amt = 8000
+
+/obj/item/weapon/rcd/syndie
+	name = "stolen rapid-construction-device"
+	desc = "A device that uses compressed matter to rapidly assemble and disassemble basic structures such as walls, flooring, and airlocks. There's something distinct about this model that screams 'tactical'."
```
Comment (ChangelingRain) [inline code/game/objects/items/weapons/RCD.dm]: This desc won't show up, because it overrides it in its New() on line 95

Comment (ghost) [inline code/game/objects/items/weapons/RCD.dm]: Oh, that's a rather stupid way to go about this item. I'll fix it later wen I get back to this PR.

**Comment by ghost** [comment]
- SYNDICATE MAGBOOT SPRITES
  ss 2014-06-08 at 06 06 51
- PINPOINTER ADDED TO UPLINK FOR TRAITORS BECAUSE FUCK THAT OBJECTIVE OTHERWISE (3TC)
- FLAMETHROWER/RCD REVERTED, NO CUSTOM PAINTJOB (TOO LAZY TO MAKE CODE WORK, SOMEONE CAN MAKE CUSTOM SYNDIE FLAMER DOWN THE LINE IF THEY WANT)
- RCD STILL HAS CUSTOM NUKE OP FILLER MAG THAT FILLS TO 30 'UNITS' OR WHATEVER THE RCD USES FOR AMMO

THIS IS TESTED ON NUKE OP AND WORKING AND SHIT

**Comment by Atskadan** [comment]
:+1:

**Comment by Giacom** [comment]
> has to steal plasma tank
> just buys it from uplink

**Pull request merged**